src/types/models/config.ts

```
@@ -3,9 +3,21 @@ export interface FlattenedProperties {
     base?: boolean;
   };
 }
+
+export interface getConvictSchemaPropertiesInputInterface {
+  encryptedValues?: boolean;
+  onlyMirroredValues?: boolean;
+  withJobHiddenProperties?: boolean;
+}
+
+export interface getConfigWithDBEncryptionStatusInterface
+  extends getConvictSchemaPropertiesInputInterface {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)